h2o-algos/src/main/java/hex/glm/GLMTask.java

```diff
@@ -1475,6 +1516,40 @@ protected void processRow(Row r) {
     }
   }
 
+  public static class GLMNegbinomialResppdate extends FrameTask2< GLMNegbinomialResppdate> {
+    private final double [] _beta; // updated  value of beta
```

Comment:
This one is used to calculate the response with new beta value and I hope to use it in theta optimization